test/integration/integration_test.cc

```diff
@@ -464,6 +464,7 @@ name: matcher
     auto response = codec_client_->makeRequestWithBody(default_request_headers_, 1024);
```

Comment:
The call to `makeHttpConnection` in line 461 should be moved here (as is done in the other test that was updated).